### DISCUSSION TITLE
Bevy only rendering blank window

### DISCUSSION THREAD

**Discussion by neoeinstein** [bevyengine/bevy#5328] [Q&A]
I pulled down the bevy repo and tried a `cargo run --example breakout`. I get sounds of the ball bouncing around, but I only get a window with a black canvas (image attached).

Mac is a 2019 MacBook Pro
CPU: 2.4 GHz 8-Core Intel Core i9
GPU: Intel UHD Graphics 630 1536 MB
On MacOS Monterey 12.4

This is my "adapter info" from the initial log message:

```
INFO bevy_render::renderer: AdapterInfo { name: "AMD Radeon Pro 560X", vendor: 0, device: 0, device_type: DiscreteGpu, backend: Metal }
```

Other examples like `sprite` also fail to produce anything on the canvas. I am wondering if there is something that I might be missing in the whole process.

**Comment by paul-hansen** [comment]
`cargo run --example breakout` is working for me.

`AdapterInfo { name: "NVIDIA GeForce RTX 3080", vendor: 4318, device: 8726, device_type: DiscreteGpu, backend: Vulkan }`

Probably a good idea to report an issue for this.
